RELEASE.md

```diff
@@ -53,7 +53,7 @@
 * The `settings.py` setting `CONF_ROOT` has been renamed to `CONF_SOURCE` to align the API. Default value of `conf` remains unchanged.
 * Renamed `extra_params` to `runtime_params` in `kedro.config.config.ConfigLoader` and `kedro.config.templated_config.TemplatedConfigLoader`.
 * Switched from packaging pipelines as wheel files to tar archive files compressed with gzip (`.tar.gz`)
-* `kedro pipeline package` and `kedro pipeline pull --alias` now accept a module path to the pipeline or utility module to package, relative to the `src/<package_name>/`.
+* `kedro pipeline package` and `kedro pipeline pull --alias --destination` now accept a module name and path to the pipeline or utility module to package, relative to the `src/<package_name>/`.
```

Comment:
`--destination` is a new option for `kedro pipeline pull` right? I would probably add a new bullet point to describe the now option and describe how the purpose of `--alias` has changed.

Reply:
Since this is develop and hasn't been released do we need to say how it has changed?

Reply:
Ahh I see what you mean, no in that case you don't have to mention how `--alias` has changed, but I would change this bullet or add another to also explain the difference between `--alias` and `--destination`